tests/unit/amo/test_store.js

```diff
@@ -29,6 +29,6 @@ describe('amo createStore', () => {
 
   it('creates an empty store', () => {
     const { store } = createStore();
-    expect(store.getState().addons).toEqual({});
+    expect(store.getState().addons.addons).toEqual({});
```

Comment:
I'm not sure what the point of this test was to be honest. I suppose you could also make it `expect(store.getState()).toEqual(initialState);`. Or maybe just delete it.

Also, I see a `describe(__filename, ...` opportunity in here.

Reply:
Yes, I figured out that we could use `initialState`...

Good catch for the `__filename` 😉